database/mysql_db.py

```diff
@@ -212,6 +193,206 @@ def get_column(self, key) -> str:
         }
         return columns.get(key)
 
-    @staticmethod
-    def db_check():
-        return"SELECT user_id FROM user;"
+    @classmethod
+    def db_check(cls):
+        return "SELECT user_id FROM user;"
+
+
+class MySQLContext(DBContext):
```

Comment:
I defined the MySQL specific context and context manager in the mysql_db module as well. so now anything that is specific to MySQL should be within this one module.